packages/manager/modules/exchange/src/account/account.controller.js

```diff
@@ -96,6 +96,7 @@ export default class ExchangeAccountHomeController {
       this.refreshList(),
     );
 
+    this.isPrivateExchange = this.productId.includes('private-ls');
```

Comment:
```suggestion
    this.isPrivateExchange = this.productId.includes('private-');
```

Reply:
I think no need to add think check
You can see in packages/manager/modules/exchange/src/dashboard/Exchange.class.js normaly this check is already done. For information private-xxxxxxx is "dedicated" offer